org.sonarlint.eclipse.ui/plugin.xml

```diff
@@ -619,6 +659,21 @@
           class="org.sonarlint.eclipse.ui.internal.command.AnalyzeCommand"
           commandId="org.sonarlint.eclipse.ui.internal.command.AnalyzeCommand">
     </handler>
+    <handler
+          class="org.sonarlint.eclipse.ui.internal.command.ExcludeCommand"
+          commandId="org.sonarlint.eclipse.ui.internal.command.ExcludeCommand">
+           <enabledWhen>
+	        <with variable="activeMenuSelection">
+	          <iterate ifEmpty="false">
+	            <or>
+                 <adapt type="org.sonarlint.eclipse.core.resource.ISonarLintFile">
+                   <test property="org.sonarlint.eclipse.core.notExcluded"/>
```

Comment:
I wonder if it is possible to revert the property, and use a negation. Like:
```
<not>
    <test property="org.sonarlint.eclipse.core.excluded"/>
</not>
```

Reply:
It is, I will try it